### PR TITLE
Ddi subunits from cp

### PR DESCRIPTION
**Pull request by abdullahhamadeh**

### CODE REVIEW

**Comment by abdullahhamadeh** [comment]
[currentReport.zip](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/files/7862655/currentReport.zip)

**Comment by Yuri05** [comment]
@abdullahhamadeh 
>currentReport.zip 

Nice! Can you attach the MD-report as well?

**Comment by Yuri05** [comment]
@abdullahhamadeh Can you please add **tidyr** to the list of installed packages?
https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/blob/76332ba6179b8c2ee7553c751661adb727d8f342/appveyor.yml#L34

I wonder why the last PR by Pierre worked - should have failed in the same way on Windows actually. Strange.

**Comment by abdullahhamadeh** [comment]
> @abdullahhamadeh
> 
> > currentReport.zip
> 
> Nice! Can you attach the MD-report as well?
[currentReportWordMD.zip](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/files/7863224/currentReportWordMD.zip)

**Comment by Yuri05** [comment]
@abdullahhamadeh It would be nice to have the images for the MD report as well 😉

**Comment by abdullahhamadeh** [comment]
> @abdullahhamadeh It would be nice to have the images for the MD report as well 😉

Yes, I added those just now

File: R/utilities-qualification.R [hunk 1]
```diff
@@ -30,6 +30,10 @@ loadQualificationWorkflow <- function(workflowFolder, configurationPlanFile) {
 
     outputsDataframeSubset <- outputsDataframe[outputsDataframe$project == project & outputsDataframe$simulation == simulationName, ]
 
+    if(nrow(outputsDataframeSubset) == 0){
+      next
+    }
+
```
[the remark this answers is not in the document]
Comment (pchelle) [reply R/utilities-qualification.R:36]: None :)

**Pull request merged**